packages/example/test/NumberUtilsPlugin.test.ts

```diff
@@ -3,6 +3,9 @@ import { expect } from 'earljs'
 describe('NumberUtilsPlugin', () => {
   it('EvenNumberMatcher works', () => {
     expect(2).toEqual(expect.evenNumber())
+
+    // it's typesafe! this is a compile time error
+    // expect('2').toEqual(expect.evenNumber())
```

Comment:
Isn't there a `@ts` annotation that will check that this is an error?

Reply:
Uhh yeah i even used it at the beginning but then realized I don't want this test to fail... I ll add another test case for this.